tests/basic/sharedv4_svc_test.go

```diff
@@ -802,16 +815,198 @@ var _ = Describe("{Shared4 service apps}", func() {
 		})
 	})
 
+	// Induce a failover while the NFS server node is in resync.
+	Context("{Sharedv4SvcResync}", func() {
+		// There are 3 replica nodes for the test-sharedv4 app's volume. We designate them as:
+		//
+		// - attachedNodeOrig: node where the volume was attached initially when the app was deployed
+		//
+		// - resyncReplica: node that will be resync'ing the volume while the volume fails over to
```

Comment:
nit: resync'ing typo?

Reply:
Apostrophe is sometimes used to indicate missing letters. In this case, it is short for resynchronizing. But "resyncing" is considered a valid word, so may be not needed. I will leave it like that so I don't have to with for the tests before merging.